storage/client/queries/queries.go

```diff
@@ -413,7 +413,9 @@ const (
 			tokens.decimals
 		FROM chain.runtime_events as evs
 		LEFT JOIN chain.address_preimages AS preimages ON
```

Comment:
Maybe a comment like this to prevent future confusion?

```suggestion
		-- Look up the oasis-style address derived from evs.body.address.
		-- The derivation is just a keccak hash and we could theoretically compute it instead of looking it up,
		-- but the implementing/importing the right hash function in postgres would take some work.
		LEFT JOIN chain.address_preimages AS preimages ON
```

Reply:
thanks; added this and updated other documentation as well